doc/package-notes.xml

```diff
@@ -219,5 +219,80 @@ you should modify
 </section>
 -->
 
+<!--============================================================-->
+
+<section xml:id="sec-eclipse">
+
+  <title>Eclipse</title>
+
+  <para>
+    The Nix expressions related to the Eclipse platform and IDE are in
+    <link xlink:href="https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/eclipse"><filename>pkgs/applications/editors/eclipse</filename></link>.
+  </para>
+
+  <para>
+    Nixpkgs provides a number of packages that will install Eclipse in
+    its various forms, these range from the bare-bones Eclipse
+    Platform to the more fully featured Eclipse SDK or Scala-IDE
+    packages and multiple version are often available. It is possible
+    to list available Eclipse packages by issuing the command:
+
+<screen>
+$ nix-env -f '&lt;nixpkgs&gt;' -qaP -A eclipses --description
+</screen>
+
+    Once an Eclipse variant is installed it can be run using the
+    <command>eclipse</command> command, as expected. From within
+    Eclipse it is then possible to install plugins in the usual manner
+    by either manually specifying an Eclipse update site or by
+    installing the Marketplace Client plugin and using it to discover
+    and install other plugins.
+  </para>
+
+  <para>
+    This installation method provides an Eclipse installation that
```

Comment:
I was confused at first because I thought the "This" in this paragraph referred to the text that _followed_. Maybe I'm wrong, but I think it is better if the sentence "This installation method provides an Eclipse installation that [...]" is moved to the previous paragraph (where the "This" actually refers to).